code/modules/surgery/organs/internal/cyberimp/augments_eyes.dm

```diff
@@ -12,8 +12,23 @@
 	name = "HUD implant"
 	desc = "These cybernetic eyes will display a HUD over everything you see. Maybe."
 	slot = ORGAN_SLOT_HUD
+	actions_types = list(/datum/action/item_action/toggle_hud)
 	var/HUD_type = 0
 	var/HUD_trait = null
+	var/toggled_on = TRUE /// Whether the HUD implant is on or off
+
+
+/obj/item/organ/internal/cyberimp/eyes/hud/proc/toggle_hud(mob/living/carbon/eye_owner)
+	if(toggled_on)
+		if(HUD_type)
+			var/datum/atom_hud/hud = GLOB.huds[HUD_type]
+			hud.hide_from(eye_owner)
+		toggled_on = FALSE
+	else
+		if(HUD_type)
+			var/datum/atom_hud/hud = GLOB.huds[HUD_type]
+			hud.show_to(eye_owner)
+		toggled_on = TRUE
```

Comment:
Shouldn't this also be removing the `HUD_trait` from the mob?

Reply:
No because this lets you still have the detailed examine info. The purpose is to clear up the screen, I think the examine info is fine.

Reply:
In that case, can you add it in the description of the action that it does that?
Something like `Disables HUD visuals. You can still access information about people you examine.`

Reply:
Done